content-src/components/Spotlight/Spotlight.js

```diff
@@ -12,14 +12,15 @@ const SpotlightItem = React.createClass({
   getDefaultProps() {
     return {
       onClick: function() {},
-      onDelete: function() {}
+      onDelete: function() {},
+      bestImage: {}
     };
   },
   render() {
     const site = this.props;
     const image = site.bestImage;
     const imageUrl = image.url;
-    const description = site.description;
+    const description = site.description || site.url;
```

Comment:
Obviously a problem for another day (like Post V1 when we get to that issue). Unless we think it'd be confusing to show URLs on only 1/2 of the results.